.travis/deploy_staging.sh

```diff
@@ -13,8 +13,8 @@ done
 # Set up group vars
 echo "setting up group vars..."
 set +x
-grep -v 'nominatim_key\|app_version:' deployment/ansible/group_vars/all.example \
-    > deployment/ansible/group_vars/all
+grep -v 'nominatim_key\|app_version:' deployment/ansible/group_vars/staging \
```

Comment:
The old version of this was essentially doing the same thing as copying `all.example` -> `all`, but on a per-variable basis. If everything is in source control it may not be necessary to do any copying at all.

Reply:
Oh, I misread the command -- this is literally copying the whole file _except_ `nominatim_key` and `app_version`, and then manually inserting `app_version` as a string literal on the next line (and `nominatim_key` and other secrets below that). The changes we're making should make copying `all.example` unnecessary, and since `group_vars/staging` can just include `app_version: latest` directly within the file, the second one shouldn't be necessary either. 

So I think these first two lines can just be deleted as long as `group_vars/staging` has `app_version` defined and doesn't define `web_js_nominatim_key`. The ones that insert secrets will need to remain though, in order to avoid committing the secrets.